src/webgpu/util/math.ts

```diff
@@ -62,3 +65,48 @@ export function diffULP(a: number, b: number): number {
   }
   return bits_a + bits_b;
 }
+
+/**
+ * @returns the next single precision floating point value after |val|,
+ * towards +inf if |dir| is true, otherwise towards -inf.
+ */
+export function nextAfter(val: number, dir: boolean = true): Scalar {
+  if (Number.isNaN(val)) {
+    return f32Bits(0x7fffffff);
```

Comment:
nit: This is NaN.   Since you already imported the kBit, it would be slightly more readable to use  kbit.f32.nan.positive here.